specification/e-referrals-service-api.yaml

```diff
@@ -165,7 +165,7 @@ info:
     Suitable values for these request headers can be obtained from the Practitioner Role Bundle returned by [[HYPERLINK_A030]].
```

Comment:
Has the Content-Type header also been included for the File Upload endpoint?

Reply:
This was one of the questions I had. I've listed them under https://nhsd-jira.digital.nhs.uk/browse/ERSSUP-60930

It makes sense to also add the Content-Type header for the file upload endpoint, but we'd have to list all media types under specification/components/schemas/file/FileContentTypeSchema.yaml. If this makes sense I'll go ahead with it.

I also wonder if we should add the content-type header to responses, and not only requests?

Reply:
Content-Type header added to request and responses, including File Upload endpoint.